packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -143,8 +143,8 @@ export class NodeHttpRequestResponseHandler extends NodeHttpStreamsHandler {
         const contentTypeHeader = response.headers['content-type'];
 
         const charsetString = contentTypeHeader ? contentTypeHeader.split(';')
-          .filter((part) => part.includes('charset='))
-          .map((part) => part.split('=')[1].toLowerCase())[0]
+          .filter((part: string | string[]) => part.includes('charset='))
+          .map((part: string) => part.split('=')[1].toLowerCase())[0]
```

Comment:
I infered types from usage here.

```suggestion
          .filter((part: string[]) => part.includes('charset='))
          .map((part: string[]) => part[0}.split('=')[1].toLowerCase())[0]
```

Reply:
Shouldn't the map still accept a string? The filter will return one string containing 'charset=...' no?